src/main/java/org/broadinstitute/hellbender/tools/walkers/vqsr/FilterVariantTranches.java

```diff
@@ -51,10 +60,38 @@
  *   --resource hapmap.vcf \
  *   --resource mills.vcf \
  *   --info-key CNN_1D \
- *   --tranche 99.9 --tranche 99.0 --tranche 95 \
+ *   --snp-tranche 99.9 \
+ *   --indel-tranche 99.0 \
+ *   -O filtered.vcf
+ * </pre>
+ *
+ * <h4>Apply tranche filters based on the scores in the info field with key CNN_2D
+ * and remove any existing filters from the VCF.</h4>
+ * <pre>
+ * gatk FilterVariantTranches \
+ *   -V input.vcf.gz \
+ *   --resource hapmap.vcf \
+ *   --resource mills.vcf \
+ *   --info-key CNN_2D \
+ *   --snp-tranche 99.9 \
+ *   --indel-tranche 99.0 \
+ *   --invalidate-previous-filters \
  *   -O filtered.vcf
  * </pre>
  *
+ * <h4>Apply several tranche filters based on the scores in the info field with key CNN_2D.</h4>
+ * This will result in a VCF with filters: CNN_2D_SNP_Tranche_99.00_99.99, CNN_2D_SNP_Tranche_99.99_100
+ * CNN_2D_INDEL_Tranche_98.00_99.00, CNN_2D_INDEL_Tranche_99.00_100
+ * <pre>
+ * gatk FilterVariantTranches \
+ *   -V input.vcf.gz \
+ *   --resource hapmap.vcf \
+ *   --resource mills.vcf \
+ *   --info-key CNN_2D \
+ *   --snp-tranche 99.0 --snp-tranche 99.99 \
+ *   --indel-tranche 98.0 --indel-tranche 99.0 \
```

Comment:
Do you want to update these recommendations based on Brian's results?  Or note that this is an example and not necessarily "best practices"?

Reply:
Added a note that the tranches may need to be tweaked. I want to do an F1 score analysis over a larger region before updating the recomendations. The first test I did  to get these numbers was just over 10m BP in chromosome 20.